src/sections/pools/navigation/Navigation.tsx

```diff
@@ -75,12 +75,11 @@ export const Navigation = () => {
         label={t("liquidity.navigation.omnipoolAndStablepool")}
         icon={<OmniStablepools />}
       />
-      {/*<Tab
+      <Tab
```

Comment:
This tab should most likely be rendered conditionally based on VITE_FF_XYK_ENABLED env variable

Reply:
will be added in the next PR, where I added that flag